src/label-for-moment/label-for-moment.test.js

```diff
@@ -3,7 +3,21 @@ import moment from 'moment';
 import _ from 'lodash';
 
 describe('_.labelForMoment()', () => {
-  it('should return in 00:00am format if the moment is in today', () => {
+  it('should return the time in the right format', () => {
```

Comment:
the previous it description was more specific/clear

Reply:
this is my new test, I still have the previous one.

Reply:
ok - I merged it, but generally best not to have extra tests if you don't need them - it's unclear from both the descriptor in the it string and from the expectations what you're testing here that's new

Reply:
makes sense. thanks